kitsune/questions/forms.py

```diff
@@ -161,6 +162,19 @@ def __init__(self, product=None, *args, **kwargs):
         )
         self.fields["title"] = title_field
 
+        if isinstance(self, NewQuestionForm) and product and product.get("categories"):
+            category_choices = [
+                (key, value["name"]) for key, value in product["categories"].items()
+            ]
+            category_field = forms.ChoiceField(
+                label=CATEGORY_LABEL,
+                choices=[("", "Please select")] + category_choices,
```

Comment:
You can `insert` the `[("", "Please select")]` right above in the `category_choices` list comprehension for better readability